lib/tarantool_server.py

```diff
@@ -590,7 +590,7 @@ def start(self, silent=True, wait=True, wait_load=True, rais=True, args=[],
                         self.current_test.is_crash_reported = True
                     color_stdout('\n[Instance "{}"] Tarantool server failed to start\n'.format(
                         self.name), schema='error')
-                    self.print_log(15)
+                    self.print_log()
```

Comment:
Don’t sure we should change log printing behaviour within the scope of this issue.

Reply:
We have logs per test case, so to investigate a failure (as described in the issue) it is good to see the whole log. And reject was just for instance.

Reply:
AFAIK we use one tarantool log per test suite (such as engine or replication) so it can get quite huge. And it will contain logs left from all test cases within the same suite, not just the failed one. I don't think it's reasonable to print it. The issue was solely about the tap test result file.